modules/builder.go

```diff
@@ -1,19 +1,17 @@
 package modules
 
 import (
-	fiftyonedegreesDevicedetection "github.com/prebid/prebid-server/v3/modules/fiftyonedegrees/devicedetection"
 	prebidOrtb2blocking "github.com/prebid/prebid-server/v3/modules/prebid/ortb2blocking"
 )
 
 // builders returns mapping between module name and its builder
 // vendor and module names are chosen based on the module directory name
 func builders() ModuleBuilders {
-	return ModuleBuilders{
-		"fiftyonedegrees": {
-			"devicedetection": fiftyonedegreesDevicedetection.Builder,
-		},
+	ret := ModuleBuilders{
```

Comment:
This file is autogenerated by https://github.com/prebid/prebid-server/blob/f3b5bcea80b2f16038ce4c53a72fb13e01e28d4d/Makefile#L23